cf-java-logging-support-servlet/src/test/java/com/sap/hcp/cf/logging/servlet/filter/SystemOutRule.java

```diff
@@ -48,4 +48,8 @@ private String getAsString(Map<String, Object> map, String key) {
 		return value != null ? value.toString() : null;
 	}
 
+    Map<String, Object> finedLineAsMapWith(String key, String expected) throws IOException {
+        return fineLineAsMapWith(key, expected);
+    }
```

Comment:
What does `fined` mean in the method name? Is this a typo, i.e. do you mean `find` or `fine`, or am I missing something?

Reply:
It's a typo.